source/extensions/omni.isaac.lab/omni/isaac/lab/scene/interactive_scene.py

```diff
@@ -128,12 +136,14 @@ def clone_environments(self, copy_from_source: bool = False):
             If True, clones are independent copies of the source prim and won't reflect its changes (start-up time
             may increase). Defaults to False.
         """
-        self.cloner.clone(
+        env_origins = self.cloner.clone(
             source_prim_path=self.env_prim_paths[0],
             prim_paths=self.env_prim_paths,
             replicate_physics=self.cfg.replicate_physics,
             copy_from_source=copy_from_source,
         )
+        if self._default_env_origins is None:
+            self._default_env_origins = torch.tensor(env_origins, device=self.device, dtype=torch.float32)
 
     def filter_collisions(self, global_prim_paths: list[str] = []):
```

Comment:
Not related to this MR but `[]` in the args is not a great practice since it is a mutable data type.

https://docs.aws.amazon.com/codeguru/detector-library/python/default-argument-mutable-objects/